client/components/FEAuth.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "../styles/FEAuth.module.css";
+import Button from "../components/Button";
+
+const FEAuth = () => {
+    return (
+        <div className={styles.main}>
+            <div className={styles.topRow}>
+                <Button
+                    text="Sign in through Steam"
+                    onClick={() => console.log("Steam Auth Stuff")}
+                />
+                <div className={styles.or}>or</div>
+                <div className={styles.profileUrl}>
+                    Enter Your Steam profile URL:
+                    <input
+                        defaultValue="https://steamcommunity.com/profiles/..."
+                        className={styles.input}
+                    ></input>
+                </div>
+            </div>
+            <div className={styles.bottomRow}>
+                Please ensure your <b>game details is set to <i >public</i></b> so we can fetch
```

Comment:
Use `<strong>` and `<em>` for bold and italic.

Reply:
Implmented but is there a reson for this? 
Is it a best practice kinda thing?

Reply:
Conveys semantic meaning beyond just bold or italic. Could be interpreted differently by accessibility tools. Use them if you're emphasising something in a sentence, otherwise `<b>` or `<i>` are fine.

Reply:
👍